ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/LoginController.java

```diff
@@ -18,7 +18,7 @@
 
 package com.netease.arctic.server.dashboard.controller;
 
-import com.alibaba.fastjson.JSONObject;
+import com.fasterxml.jackson.databind.JsonNode;
```

Comment:
Do we need to import the shaded class here?

Reply:
yes, we should use the shaded class